WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/Downloadable Files/ProductDownloadFileViewModelTests.swift

```diff
@@ -0,0 +1,297 @@
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+final class ProductDownloadFileViewModelTests: XCTestCase {
```

Comment:
Can we conform all these class naming to `DownloadableFile`, like `ProductDownloadableFileViewModelTests`?

Reply:
Sorry @pmusolino , I actually didn't understand what you have suggested here. Can you please elaborate?

Reply:
We are using different class naming for Downloadable Files, so I think that we should update the name of these classes, for example changing  `ProductDownloadFileViewModelTests` to `ProductDownloadableFileViewModelTests` and so on.

Reply:
yes. I have changed all occurrences of `DownloadFile` and `Downloads` to `DownloadableFile`